ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/InMemoryKVParticipantStateIT.scala

```diff
@@ -387,7 +389,7 @@ class InMemoryKVParticipantStateIT
 
 object InMemoryKVParticipantStateIT {
   private val emptyTransaction: SubmittedTransaction =
-    PartialTransaction.initial.finish.right.get
+    GenTransaction(SortedMap.empty, ImmArray.empty, Some(InsertOrdSet.empty))
```

Comment:
What exactly is the difference between passing in an empty set and passing `None`?

Reply:
I wanted to be explicit that the package dependencies were not even computed.